pkg/controller/masteruserrecord/sync_test.go

```diff
@@ -131,7 +124,7 @@ func TestSynchronizeSpec(t *testing.T) {
 	userAccount := uatest.NewUserAccountFromMur(mur)
 
 	murtest.ModifyUaInMur(mur, test.MemberClusterName, murtest.NsLimit("advanced"),
-		murtest.TierName("admin"), murtest.Namespace("ide", "54321"))
+		murtest.TierName("admin"), murtest.Namespace("basic-dev-123abc", "admin-ide-54321"))
```

Comment:
same here, you are replacing with different tier and with a different namespace type

Reply:
fixed as part of: 000a2b5594241a958671b11904043853eb3fbb47